tests/manage/mcg/test_bucket_creation.py

```diff
@@ -3,14 +3,16 @@
 import pytest
 
 from ocs_ci.framework.pytest_customization.marks import (
-    tier1, noobaa_cli_required, aws_platform_required, filter_insecure_request_warning
+    tier1, noobaa_cli_required, aws_platform_required,
+    filter_insecure_request_warning, acceptance
 )
 
 logger = logging.getLogger(__name__)
 
 
 @filter_insecure_request_warning
 @aws_platform_required
+@acceptance
 @tier1
 class TestBucketCreation:
```

Comment:
Just FYI: based on the results in Jenkins, this test newer passed.

Reply:
In the last run, I do see them as passed, but in earlier runs it was skipped, we can alway remove or add mark `@acceptance` based tier1 stability.
```

Passed | tests/manage/cluster/cluster_expansion/test_node_expansion.py::TestAddNode::test_add_node | Test for adding worker nodes to the cluster while IOs | 275.03
-- | -- | -- | --

Passed | tests/manage/mcg/test_bucket_creation.py::TestBucketCreation::test_s3_bucket_creation | Test bucket creation using the S3 SDK | 0.41

Passed | tests/manage/mcg/test_bucket_creation.py::TestBucketCreation::test_cli_bucket_creation | Test bucket creation using the MCG CLI | 1.80

Passed | tests/manage/mcg/test_bucket_creation.py::TestBucketCreation::test_oc_bucket_creation | Test bucket creation using OC commands | 1.83


```